Developed homepage with styling

Adams-Ghub opened the pull request:
- Developed the homepage in HTML format.
- Developed a stylesheet for the homepage to style it.
- Created a Markdown file called test.

Adams-Ghub committed:
created homepage.html file and added an h1 tag to it

Adams-Ghub committed:
created a stylesheet for the homepage called homepage-styles.css

Adams-Ghub committed:
created the test md file and added it to the gitignore file

Ibizugbe commented on .gitignore:
```diff
@@ -1,2 +1,3 @@
 # .gitignore
-node_modules/
\ No newline at end of file
+node_modules/
+test.md
```
Nice work creating the `.gitignore` file and adding the necessary files👍 but these files haven't been created🥲 You currently do not have the `node_modueles` folder because you haven't' setup linters for the project🥲 also, you haven't set up the `test.md` file as required😉 kindly take a look at the [requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)and make the necessary change😉

your current directory
<img width="303" alt="filedirectory_notest_nonodemodules" src="https://user-images.githubusercontent.com/64643995/219342556-429bd50f-c7ef-4c52-8267-389d4e5ba913.png">

setup test.md instruction
<img width="391" alt="testgitignore" src="https://user-images.githubusercontent.com/64643995/219342908-377792ce-e50e-474e-8030-54534e91da8a.png">

Ibizugbe reviewed (changes requested):
Hi @Adams-Ghub 👋, 

![198009982-d99c2507-f20d-4c00-90a3-a54097598b36](https://user-images.githubusercontent.com/64643995/219343374-32cedc43-c30f-406a-931f-c55c5b60060b.gif)


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Pull request has been made from the `feature` branch to the `main` branch👍👍.
- [x] `HTML` and `CSS` file has been added👍.


## Required Changes ♻️

- [ ] Currently, you haven't setup linters for your project, and it is a[ requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that you set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git. kindly fix this🤗
- [ ] Nice job setting up the readme with the microverse readme template, but don't forget, the template is just a guide line to aid you in setting up a professional readme and not an alternative😉 kindly fill all the sections in the template with the information necessary to this project🤗🤗😉 as it is a [requirement](Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.) that you add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] Your Pull reqyest doesn't have a descfription and this can give your repository a less professional look😉 kindly add a descriptive summary to your project and it should give details about the features or changes you have added to the branch👍👍 [remember](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), it is a requirement that our project is [professionally documented](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)🤗🤗
also, I can see you added a heading to your pull request but I think it is not descriotive enough😉 we can make it more descriptive by ensuring it describes or relates to the changes made in the pull request🤗🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Adams-Ghub committed:
configured linters properly

Adams-Ghub committed:
Created a LICENSE.md file

Used MIT LICENSE Template to create a License file

Adams-Ghub committed:
updated the README.md file

Adams-Ghub committed:
Updated the README.md file better

Adams-Ghub committed:
Merge branch 'added-html-css-md-files' of https://github.com/Adams-Ghub/HelloMicroverse into added-html-css-md-files

Adams-Ghub committed:
Updated the README file

Changed the path to the license file

Adams-Ghub committed:
Updated the README file

Added a .md extension to the path to the license file

iambenkis commented on README.md:
```diff
@@ -247,41 +90,15 @@ Feel free to check the [issues page](../../issues/).
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](/LICENSE.md) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ] [ **OPTIONAL** ] Kindly make sure you remove this Note text , it will make your README file more professional :+1:

iambenkis commented on README.md:
```diff
@@ -25,37 +25,16 @@ OPTIONAL SECTIONS:
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
```
- [ ] Nice job setting up the README with the microverse README template, as requested by the previous code reviewer  you should  fill all the sections in the template with the information necessary to this project as it is a [requirement](Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.) that you add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. Kindly fix that to go to the next project :+1:

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Adams-Ghub 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 

- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Documentation :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Adams-Ghub committed:
Updated the README file

Added Contributing and Acknowledgement sections.
Removed all text that gave the README file an unprofessional look.

franclobo commented on README.md:
```diff
@@ -1,288 +1,79 @@
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
```
- [ ] As requested by the previous code reviewers All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo. This is mandatory for al the projects onwards otherwise they will be invalidated.

| Expected | Yours |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219669304-8375dfad-2850-49c5-a7d5-fe0231731e11.png) | ![image](https://user-images.githubusercontent.com/58642949/219669588-671a90b8-e8d1-40ac-b9cf-146496b4d022.png) |

Adams-Ghub replied:
Hello @franclobo, I really appreciate the review you have done on my project. I want to clarify something.
Do I have to bring the full table of content and write under every main header in the README file even for those that may not be applicable to my project? or do I have to bring the table of content and optionally leave out their description in the README file

franclobo commented on README.md:
```diff
@@ -1,288 +1,79 @@
 
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
```
- **[OPTIONAL]** This was not explicit by previous code reviewers but In order to follow the README template instructions, please 🙏🏽 remove the guidelines for professional documentation.

franclobo reviewed (changes requested):
Hi @Adams-Ghub, 

# ♻️ Required Changes ♻️

![soClose](https://user-images.githubusercontent.com/58642949/219671657-6d3c94c2-183d-4b49-9703-3c818902b464.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Adams-Ghub committed:
Updated README file

worked on the table of contents

Adams-Ghub committed:
modified the README file

Edited the table of contents
Edited the get started section

victorokoroji reviewed (approved):
Hi @Adams-Ghub, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
